region/maridia/inner-pink.json

```diff
@@ -6584,6 +6630,46 @@
                     "Bouncing on the sand as springball can reach the above maze by altering Samus' fall height, like by bouncing under the solid ledge before trying to jump up."
                   ],
                   "devNote": "Jumping through the sandfall is failable enough to require canPlayInSand"
+                },
+                {
+                  "name": "Shinespark",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateUnderwater",
+                    "canPlayInSand",
+                    "canShinechargeMovement",
+                    {"or": [
+                      {"and": [
+                        {"or": [
+                          "canResetFallSpeed",
+                          {"and": [
+                            "canPrepareForNextRoom",
+                            "h_canUsePowerBombs"
```

Comment:
I'm assuming this is for the shot blocks - I'm not sure how reset fall speed helps with this. should it be power bombs and reset fall speed or prepare for next room?

Oh, I see the comment now - can't you shoot upwards with both angle buttons? Should this strat require a hero shot?